ui/pages/asset/asset.scss

```diff
@@ -1,13 +1,52 @@
 @use "design-system";
 
+
+
 .asset {
+  $max-width: 650px;
+
+  &__wrapper {
+    @media screen and (max-width: design-system.$screen-sm-max) {
+      width: 100%;
+    }
+  }
+
+  &__header {
+    min-height: 71px;
+  }
+
   &__container {
     background-color: var(--color-background-default);
   }
 
+  &__content {
+    max-width: $max-width;
+  }
+
+  &__footer {
+    position: sticky;
+    bottom: 0;
+    box-shadow: 0 0 8px 0 var(--color-shadow-default);
+    & > :first-child { max-width: $max-width; }
+  }
+
   &__overview {
     box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
   }
+
+  &__chart {
+    &--no-data {
+      background:
+        linear-gradient(0deg, rgba(255, 255, 255, 0.2), rgba(255, 255, 255, 0.2)),
+        linear-gradient(102deg, #e7f2f8 39.39%, #ebf2fa 50.57%, #eff2fc 61.75%, #f3f2ff 95.3%);
```

Comment:
Summoning @georgewrmarshall  about these rogue hex values.  I'm matching the subtle gradient in the [design](https://www.figma.com/file/TfVzSMJA8KwpWX8TTWQ2iO/Asset-list-and-details?node-id=2515%3A89742&mode=dev) below.  Any concerns?


<img width="420" alt="Screenshot 2024-03-07 at 11 19 43 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/562c1fb8-79f3-4d84-9cd3-275506a9be58">